API_CHANGES.md

```diff
@@ -3,11 +3,11 @@
 ## Breaking changes
 
 ## New features
-* Add a set of methods to `org.sonarsource.sonarlint.core.rpc.protocol.backend.analysis.AnalysisRpcService` the service to let the client notify the backend with analysis properties - inferred by SonarLint and defined by the user
+* Add a method to `org.sonarsource.sonarlint.core.rpc.protocol.SonarLintRpcClient` to allow backend to request inferred analysis properties from the client before every analysis. It's important because properties may change depends on files being analysed and to spare clients from implementing listeners of configuration change.
```

Comment:
`may change depends` -> `may change depending on`.
I would also remove the last part "and to spare...", this tells more about the design decision that the actual change clients need to be aware of.

Reply:
Changed